### DISCUSSION TITLE
ASM and CAM metric

### DISCUSSION THREAD

**Discussion by ehsankf** [Trusted-AI/adversarial-robustness-toolbox#1412] [Q&A]
Hello,

I came across this paper https://openreview.net/pdf?id=BkgzniCqY7 from IBM.
I wonder if the metrics such as CAM and ASM introduced in this paper are integrated
into the ART.

Thanks.

**Comment by beat-buesser** [comment]
Hi @ehsankf Thank you for your question. These tools have not yet been integrated to ART, but I think they would be very useful and great to have integrated.

Do you have plans to implement them?
